tests/data/raml_10/data-type-examples.raml

```diff
@@ -0,0 +1,150 @@
+#%RAML 1.0
+
+title: Example
+baseUri: https://example.test/{param}/
+
+baseUriParameters:
+  param:
+    type: string
+    description: account name
+    example: splat
+
+mediaType: application/json
+protocols: [ HTTPS ]
+
+types:
+
+  with_multiple_examples:
+    type: string
+    examples:
+      simple: "abc"
+      fancy: "two words"
+      excessive:
+        displayName: "Serious Overkill"
+        description: |
+          There's way too much text here.
+        value:
+          error: This should not be a map.
+        strict: false
+
+  with_example_structured:
+    type: object
+    properties:
+      key:
+        type: string
+        example:
+          description: Typical value.
+          value: This is a value.
+    example:
+      description: This is a typical structure.
```

Comment:
Don't these keys need to map to keys defined in `properties?` Looking at [this example](https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#example-of-how-to-define-exampleexamples-in-raml) in the spec:

```raml
properties:
  name: string
  lastname: string
example:
  name: Bob
  lastname: Marley
```